src/servicebus/azext_servicebus/_params.py

```diff
@@ -0,0 +1,408 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands.parameters import (tags_type, get_enum_type, resource_group_name_type, name_type)
+
+from azure.cli.command_modules.servicebus._validators import _validate_max_delivery_count, _validate_maxSizeInMegabytes,\
+    _validate_dead_lettering_on_message_expiration, _validate_requires_session, _validate_enable_batched_operations, \
+    _validate_support_ordering, _validate_enable_express, _validate_enable_partitioning,\
+    _validate_dead_lettering_on_message_expiration,\
+    _validate_requires_session, _validate_requiresDuplicateDetection, _validate_auto_delete_on_idle, \
+    _validate_duplicate_detection_history_time_window, _validate_default_message_time_to_live, _validate_lock_duration
+
+
+# pylint: disable=line-too-long
+def load_arguments(self, _):
+
+    # region - Namespace Create
+
+    with self.argument_context('sb namespace check_name_availability') as c:
+        c.argument('namespace_name', options_list=['--name'], required=True, help='name of the Namespace')
+
+
+    with self.argument_context('sb namespace create') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type, required=True)
+        c.argument('namespace_name', options_list=['--name'], required=True, help='name of the Namespace')
```

Comment:
You shouldn't need this `required=True` as CLI will automatically detect if it's required from the method signature of the command.

Reply:
Same elsewhere

Reply:
resolved